bcgov/HOUS-permit-portal
# Feature/bphh 1514/api mappings setup

## Description
- Refactor naming from jurisdiction_integration_requirements_mapping to integration_mapping to reduce verbosity
- Adds api mapping for local systems for a jurisdiction


## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature

## Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-1514


## Steps to QA
Pre requisite:
- Ensure api is enabled for any jurisdiction of choice
Steps:
- Log in as review manager
- Navigate to api-settings page over top right menu
- Click "API mappings setup" to navigate to page for QA
- Should be able perform the following actions on this screen
  - Edit mapping
  - Filter by unmapped values
  - search
  - collapse 
![Screenshot 2024-06-14 at 12 01 00 PM](https://github.com/bcgov/HOUS-permit-portal/assets/32022335/f98e2586-eb50-4501-a93f-1c879cfc34b8)
